src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -423,7 +423,49 @@ def test_WINNF_FT_S_REG_5(self):
     for x in range(1, 4):
       self.assertEqual(
           response['registrationResponse'][x]['response']['responseCode'], 102)
+  
+  @winnforum_testcase
+  def test_WINNF_FT_S_REG_8(self):
+    """Invalid REG-Conditional parameters in Array Registration Request (responseCode 103)
+
+    The response should be SUCCESS for the first CBSD,
+    FAILURE 103 for the second and third CBSDs.
+    """
+
+    # Load devices
+    device_1 = json.load(open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_3 = json.load(open(os.path.join('testcases', 'testdata', 'device_e.json')))
+
+    # Inject FCC IDs
+    self._sas_admin.InjectFccId({'fccId': device_1['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_2['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_3['fccId']})
 
+    # Inject User IDs
+    self._sas_admin.InjectUserId({'userId': device_1['userId']})
+    self._sas_admin.InjectUserId({'userId': device_2['userId']})
+    self._sas_admin.InjectUserId({'userId': device_3['userId']})
```

Comment:
it looks good, only one optional tiny comment, isn't it shorter to inject User Id and Fcc Id by a loop through the devices?

Reply:
probably....I think we were just modeling things after the way earlier TCs were impemented

Reply:
yes, no problem then :)